### ISSUE TITLE
Extend node-style-guide?

### ISSUE THREAD

**Issue by pdehaan** [gulpjs/eslint-config-gulp#5]
We have [**node-style-guide**](https://www.npmjs.com/package/node-style-guide) and [**eslint-config-node-style-guide**](https://github.com/pdehaan/eslint-config-node-style-guide/blob/master/index.js) to choose from.

But it may be good to use **node-style-guide** as a base set of rules and then define our overrides in **eslint-config-gulp**. It'd make it easier to track that we've overridden the `"object-curly-spacing": [2, "always"]` rule (or any other future rules).

Basically, `npm i eslint-config-node-style-guide -D` and then set our eslint.json like:

``` js
{
  "extends": "node-style-guide",
  "rules": {
    "object-curly-spacing": [2, "always"]
  }
}
```

**Comment by phated** [comment]
:+1: now that you are a maintainer of node-style-guide.  Something is broken in the PR though.

**Comment by phated** [comment]
Closing this as the accompanying PR was closed.
